circle.yml

```diff
@@ -5,6 +5,11 @@ general:
       # then we are prepping for release an there is no need to build it again.
       - master
 
+checkout:
+  post:
+    # Update the `master` branch. Otherwise CircleCI will give us a cached one.
+    - git fetch -u origin +master:master
```

Comment:
Should note this is in contrast to [build]( https://circleci.com/gh/conda-forge/staged-recipes/8394 ), which is using the cached repo. Note `master` includes `vtk`, which was [there]( https://github.com/conda-forge/staged-recipes/commit/e8c8e9f3376533c9281819976ade9eb2b9145076 ) and then [removed]( https://github.com/conda-forge/staged-recipes/commit/8387f946979462901b0930b669e21ee923f6bbe8 ) a couple of months back. Hence why we need to refresh the repo.